### ISSUE TITLE
Running VisualPHPUnit 3.x

### ISSUE THREAD

**Issue by bobc82** [VisualPHPUnit/VisualPHPUnit#162]
I tried to configure VisualPHPUnit 3.x following these steps:

[https://github.com/VisualPHPUnit/VisualPHPUnit#setup](https://github.com/VisualPHPUnit/VisualPHPUnit#setup)

and it seems that configurations was successful.

When i run the command ./bin/vpu -c vpu.json -s i think that built in server starts, so i can view 'VPU started' message after i run that command.
But when i try to access at http://localhost:8000 from my web browser i don't see anything, only a failed connection message. I'd like to know what is missing. Thank you in advance!
(in the past i used VisualPHPUnit 2.x and it runs fine!)

**Comment by c0h1b4** [comment]
same issue here. Also, nginx setup is not correct, so anyone please review it.

**Comment by HeindeRuiter** [comment]
I've solved this by renaming the `docroot` for the frontend in  `src/Console/Command/Run.php` from `dist` to `app`.  

```php
<?php
'frontend' => [
                'host' => 'localhost',
                'port' => 8000,
                'docroot' => 'app'   //change this line from dist to app
            ],
```
Then the `bower components` wouldn't load so I had to move those to the `app` folder.

**Comment by bobc82** [comment]
The solution of HeinDR works for me! Thanks!

**Comment by HeinDR** [comment]
I've found a better solution. The dist folder should be created by Grunt. The documentation does not include the step running Grunt. Only for development. Grunt however fails on errors. To solve this run: ```grunt --force```

**Comment by localgod** [comment]
Documentation and grunt build updated. Documentation should be clearer and grunt should run without the need of `--force`.

**Comment by itsDiwaker** [comment]
I am having similar issue. Here's what I did:
Downloaded the package from http://visualphpunit.github.io/VisualPHPUnit/.
Ran `make tools setup build`
Then on command line I did `./bin/vpu -c vpu.json -s`
It showed VPU started.
Pointed browser to localhost:8000 but it said connection refused.

Tried to debug on my own and got below observations:
On terminal doing grunt serve opens the app on port 10000 without any problem.
Copying `bower_components` to `app` folder and changing the docroot in  `src/Console/Command/Run.php` from `dist` to `app` works for me.
After doing above steps I was only able to load front-end only after I did `php -S localhost:8000` in `app` folder. Is this was required but not mentioned in the documentation explicitly?

Why I can't see the `dist` folder?
